### ISSUE TITLE
Video freezing when using Video Speed Controller

### ISSUE THREAD

**Issue by kintosh** [igrigorik/videospeed#51]
Hello there!

First of all I wish to say I LOVE your app :)

That said, I wish to know if you are aware of an issue that when using video speed controller the video freezes? (it freezes, unfreeze, freeze and unfreeze again and keep doing it) Tried reinstalling but didn't help. 

Thank you very much and I'be very glad for your reply.
Take care!

**Comment by Wisdawn** [comment]
I can second this experience; it happens to me with some videos occasionally. Unfortunately, I didn't think it was important enough to report and to link specific examples, and I thought that the examples won't show the same behavior again on another computer, because maybe it was from my machine; but now that another user reports it, perhaps it's not my machine, after all.

@kintosh I recommend that you link specific examples of videos that behave that way for you. I'll try and watch them, too, if I have time, and confirm whether they behave similarly on my end.

**Comment by kintosh** [comment]
Hello there! This vid for example freenzes in 1.2,   1,5x or above: https://www.youtube.com/watch?v=_B0SMOlGdh8

Thanks for your reaply! :)

**Comment by igrigorik** [comment]
I don't think this has anything to do with the extension itself.. it's sounds like a buffering issue (i.e. function of your connection speed). I don't have any issues watching that video in any of those speeds. Have you tried selecting lower quality video?

**Comment by Wisdawn** [comment]
I second the feedback from @igrigorik. I can also watch that video without problems. And it's true; I remember that when I had that issue with a few videos, lowering the quality helped sometimes, but not always.

With that said, some low quality videos still stuttered. I also remember that when I brought the speed back to 1.0, no stuttering occurred. So there's still the small chance that the extension may be related, even if indirectly in relation to processing power or so.

**Comment by igrigorik** [comment]
@Wisdawn it's "related" in the sense that we're taxing the CPU; we're downloading more data and crunching through more frames. The browser doesn't (and can't) skip frames, so playing at higher speed requires more bandwidth and CPU resources. As such, yes, it may "cause" stuttering, but it's not anything we can fix in the extension itself.

**Comment by Wisdawn** [comment]
@igrigorik That does make sense to me, yes. That's partially why I had a feeling that it was not something that I needed to report when I experienced that stuttering sometimes.

@kintosh As igrigorik said, try lowering the video quality; and failing that, try restarting the computer and minimizing what's running in the background or simultaneously; like minimize open tabs, applications, and so on. That helped me in the past sometimes.
